client/client.go

```diff
@@ -200,7 +200,11 @@ func (client *Client) ServerVersion() (string, error) {
 		version = "unknown"
 	}
 
-	return fmt.Sprintf("%s (series %s)", version, sysInfo.Series), nil
+	if sysInfo.OnClassic {
+		return fmt.Sprintf("%s (series %s, os: %s, classic)", version, sysInfo.Series, sysInfo.OS), nil
```

Comment:
Done. 

Note that there's a FIXME for getting the snap (client) version at compile time. I need to check how this is done for snapd and apply the same logic.